decidim-elections/spec/helpers/decidim/elections/votes_helper_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Decidim
+  module Elections
+    describe VotesHelper do
+      describe "ordered_answers" do
+        subject(:uniq_results) { repetitions.times.map { helper.ordered_answers(question) } .uniq }
+
+        let(:repetitions) { 100 }
+        let(:question) { create :question, :complete, answers: 3 }
+
+        it "orders answers in different order on different calls" do
+          # with 100 repetitions, this test could result on a false negative on 1 of 13.802.995 executions: 1.0/(6*(5.0/6)**100)
```

Comment:
I planned to ask about this 😅 
What do you think that it would be better?
```ruby
# with 3 answers, there are 3! = 6 possible ways to reorder them
# for each ordering way, the probability of not being used is 5/6**N, where N is the number of repetitions of the process 
# the probability of not generating all possible orders is the sum of not use each ordering way, 6*(5/6**N)
# with 100 repetitions, this test could result on a false negative on 1 of 13.802.995 executions: 6*(5.0/6)**100
```
or
```ruby
# This test could randomly fail with a very very very low probability (1 of 13.802.995 times)
```

Reply:
😀 The first option explains why it could fail, the second one warns you.
Don't have a strong opinion about it, maybe the second one is more concise.
Thanks